logverification/app/assetsv2.go

```diff
@@ -33,14 +34,14 @@ func NewVerifiableAssetsV2Events(eventsJson []byte) ([]VerifiableAssetsV2Event,
 		return nil, err
 	}
 
-	events := []VerifiableAssetsV2Event{}
+	events := []*AssetsV2AppEntry{}
```

Comment:
Why a slice of pointers?

Generally considered an anti-pattern unless you explicitly want to test for nil.

See https://medium.com/@philpearl/bad-go-slices-of-pointers-ed3c06b8bb41

for a discussion

Reply:
also you can specify a size up fron - len(eventListJson.Events) which is much more efficient

Reply:
ahh can do :)

Reply:
just return the list of interface type now :) is that acceptable?

Reply:
I do not know if it suffers from the same performance problems?